index.bs

```diff
@@ -391,58 +391,30 @@ See also:
 
 <h3 id="private-browsing-mode">Consider how your API should behave in private browsing mode</h3>
 
-In general,
-APIs should behave the same in and out of [private browsing mode](https://www.w3.org/2001/tag/doc/private-browsing-modes/).
-This is because
-the use of private browsing mode should not be revealed to sites.
-See [[#do-not-expose-use-of-private-browsing-mode]].
```

Comment:
I think it would be good to retain some reference to the other section, since somebody might be linked here directly.

Reply:
Added to "See also" links at the bottom.